src/d3kit-gridmap.js

```diff
@@ -6,6 +6,7 @@ var DEFAULT_OPTIONS = {
   initialWidth: 500,
   initialHeight: 340,
   keyFn: undefined,
+  durationFn: function() {return 250;},
```

Comment:
Perhaps rename this to transitionDuration to reduce name ambiguity and just set it to 250. No function needed.

```
transitionDuration: 250
```

Also update the documentation in `docs/api.md` would be nice.